service/pokedex.go

```diff
@@ -0,0 +1,72 @@
+package service
+
+import (
+    "os"
+    "fmt"
+    "strings"
+    "strconv"
+
+    "pokedex/config"
+    "pokedex/model"
+    "pokedex/model/errors"
+    "pokedex/data"
+)
+
+/* This map will act as an in-RAM Pokedex */
+type Pokedex map[int]model.Pokemon
+
+/* Parses data read from CSV into a Pokedex map so it can be consumed later */
+func (pokedex Pokedex) Init() {
```

Comment:
Instead of having an Init function, let's better use Clean architecture best practices.

Your project should be separated by its responsibilities (layers), which you already have.

These layers should be represented by a struct, and as fields, you should add all dependencies your layer might require.

Also, to have a New<layerName> function, to be able to return an object of your layer.

As an example here in the service, you should opt for having something like this, instead of the Init approach:

```
type Service struct {
	source string
}

func NewService(source string) *Service {
	s := &Service{
		source: source,
	}
	
	return s
}
```

The idea of this is to first have initialized all your layers from the main, and passing all values there, avoiding doing tasks that are not part of the layer's responsibilities; instead of having your service reading config values, you pass it from main.

Please, give it a look to an example I shared in the channel: https://github.com/RolloCasanova/sample-clean

And another look at what a Go project with clean architecture might look like. 
https://eltonminetto.dev/en/post/2020-07-06-clean-architecture-2years-later/